recipes/toad/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "toad" %}
+{% set version = "0.0.58" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9dce218ac11eb9e3a80b233557694d0d349418a1ba517306919dff02595e5f07
+
+build:
+  number: 0
+  skip: True  # [py==38 or py==35 or py==27]
+  entry_points:
+    - toad = toad.cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy >=1.15.4
+    - cython
+    - setuptools
+  run:
+    - python
+    - numpy >=1.15.4
+    - pandas
+    - scipy
+    - scikit-learn
+    - statsmodels >=0.10
+    - seaborn
+
+test:
+  imports:
+    - toad
+
```

Comment:
If `toad -h` works, then the test should be:
```
  commands:
    - toad -h
```
or whatever command would return exit code 0.